src/olympia/editors/models.py

```diff
@@ -126,22 +126,19 @@ def base_query(self):
             ]),
             'from': [
                 'addons',
-                """JOIN (
-                    SELECT MAX(id) AS latest_version, addon_id FROM versions
-                    GROUP BY addon_id
-                    ) AS latest_version
-                    ON latest_version.addon_id = addons.id
-                LEFT JOIN versions
-                    ON (latest_version.latest_version = versions.id)
-                JOIN files ON (files.version_id = versions.id)""",
+                """
+                LEFT JOIN versions ON (addons.id = versions.addon_id)
+                LEFT JOIN files ON (files.version_id = versions.id)
 
-                #  Translations
-                """JOIN translations AS tr ON (
-                            tr.id = addons.name
-                            AND tr.locale = addons.defaultlocale)"""
+                JOIN translations AS tr ON (
+                    tr.id = addons.name
+                    AND tr.locale = addons.defaultlocale)
+                """
             ],
             'where': [
                 'NOT addons.inactive',  # disabled_by_user
+                'versions.channel = %s' % amo.RELEASE_CHANNEL_LISTED,
+                'files.status = %s' % amo.STATUS_AWAITING_REVIEW,
                 # Are we showing listed or unlisted addons?
                 '{0} addons.is_listed'.format('' if self.listed else 'NOT'),
```

Comment:
should probably either take this out or at least make it consistent with the `versions.channel` line above